pyproject.toml

```diff
@@ -8,9 +8,9 @@ packages = [{ include = "fastapi_keycloak_middleware" }]
 
 [tool.poetry.dependencies]
 python = "^3.10"
-fastapi = ">=0.73.0"
+fastapi = ">=0.110.0"
 install = "^1.3.5"
-python-keycloak = ">2.14.0,<4"
+python-keycloak = "^3.9.1"
```

Comment:
We should make sure that we support newer versions within the same major version. I'd prefer `">3.9.1,<4"` instead

Reply:
"^" does exactly the same, because it won't allow changes to the left most non-zero digit in the version tag (see [Poertry documentation](https://python-poetry.org/docs/dependency-specification/#caret-requirements)). If you prefer the explicit declaration instead, I would change it to ">=3.9.1, 4" or ">3.9.0,4" because 3.9.1 is the newest version 😅

Reply:
There's times I should stick to sleeping... you're right of course, that's fine.